### PR TITLE
Support :prepend and :append for the `select` helper

### LINKED ISSUE #147: Support :append for the `select` helper
Is there an easy way to append label to form select helper?  The following does not seem to work...

```
<%
    options = [['first', 1]]
    (2..30).each {|day| options << [day.to_s, day]}
    options << ['last', 31]
%>
<%= f.select :due_day, options, append: 'day', label: 'Due Day' %>
```

### PR DESCRIPTION
**Pull request by donv**
Fixes #147

### CODE REVIEW

File: test/bootstrap_selects_test.rb [hunk 1]
```diff
@@ -32,6 +32,20 @@ def setup
...
+  test 'selects with addons are wrapped correctly' do
+    expected = <<-HTML.strip_heredoc
+      <div class="form-group">
+        <label class="control-label" for="user_status">Status</label>
+        <div class="input-group">
+          <span class="input-group-addon">Before</span>
+          <select class="form-control" id="user_status" name="user[status]"><option value="1">activated</option>
+      <option value="2">blocked</option></select>
+          <span class="input-group-addon">After</span>
+        </div>
```
Comment (mattbrictson) [inline test/bootstrap_selects_test.rb]: Is there another `</div>` needed here?

Comment (donv) [inline test/bootstrap_selects_test.rb]: Yes, absolutely.  I added the </div> and structured the HTML example a little better as well.

**Comment by mattbrictson** [comment]
Awesome, thank you 🙌

**Comment by donv** [comment]
@mattbrictson How do we propagate this to the bootstrap-v4 branch?

**Comment by mattbrictson** [comment]
@donv Can you create a new branch from the latest commit of `bootstrap-v4` and then cherry pick  commit 0fba844 ? Then when you open a PR, make sure the base branch of the PR is set to `bootstrap-v4`.

That's assuming the cherry-pick goes cleanly. Let me know if it doesn't work out.

**Comment by donv** [comment]
@mattbrictson OK, will do.

**Comment by donv** [comment]
@mattbrictson OK, added https://github.com/bootstrap-ruby/rails-bootstrap-forms/pull/327

**Pull request merged**